.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,55 @@
+name: Nightly
+on:
+  pull_request:
```

Comment:
It runs on every pull requests?

Add "workflow_dispatch" so that we can invoke the job manually.

Reply:
But also:
> To trigger the workflow_dispatch event, your workflow must be in the default branch.

So `pull_request` has been useful to develop this in the PR, but I'm now replacing with `workflow-dispatch`.